src/threading.ts

```diff
@@ -103,6 +104,10 @@ class Thread {
         this.stack.push(frame);
     }   
 
+    public tracebackClear(): void {
```

Comment:
I'd prefer the name `clearTraceback()`, because it's more consistent with `addToTraceback()`.